### DISCUSSION TITLE
Renault Laguna 3 left / right mirror parking position

### DISCUSSION THREAD

**Discussion by Uranus-UI** [cedricp/ddt4all#1306] [General]
Does anyone know how to set the mirror to automatically tilt when parking?

**Comment by AndyIreland** [comment]
on Clio Mk4 its in the UPC-EMM ECU . Cannot see that setting in your UPC-EMM for the Laguna so dare say your car is not new enough for AutoFolding setting.

**Reply by Uranus-UI** [reply]
Hi, thank you for your reply. I don't know if I have correctly described what I need. In the link there is a video feature that I would like to activate on my Laguna.
https://youtu.be/OrpnbSEYDcs?feature=shared

I know that the Laguna has this feature in some equipment. I wondered if it could be activated somewhere.

**Reply by AndyIreland** [reply]
ah right, no sorry I was replying to your original post which said you wanted your mirrors to fold.
